libgalois/include/katana/PropertyFileGraph.h

```diff
@@ -419,6 +471,8 @@ class KATANA_EXPORT PropertyFileGraph {
 
   bool empty() const { return topology().empty(); }
 
+  // NB: num_nodes in repartitioner is of type LocalNodeID
+  /// Return the number of local nodes and edges
```

Comment:
I think this note also applies to size. So it should be there too. Also please make this into a doc comment. It's pretty important for API users.

Reply:
I flipped the lines and made the comment about num_nodes a doc comment.  At no additional charge, I removed the NB: prefix that I so dearly love.  I'm not sure what "So it should be there too" means.

Reply:
The `size` method returns the same value as `num_nodes`. So it too will be only local nodes in some cases. that's all I meant. The way you rewrote the comments makes me think I may have misunderstood in the first though though.

In the distributed case, is `num_nodes` (and `size`) the total number of global nodes in the graph, or is it the number of local nodes present in the current process?

Reply:
So you want me to add a comment to PropertyFileGraph::size that it only returns the number of local nodes.  I didn't notice that size returns num_nodes.  I can add this.

The comment is trying to say that num_nodes returns the number of local nodes and edges.  That is not the number of global nodes in the distributed case.  Do you have an improved way to phrase the comment?

Reply:
"Returns the number of local nodes" is just fine. Has it said that from the beginning? If so, sorry. I just failed to read it right and none of this was an issue.

Just a nit, but doc comments only associate with the function below them, so if you want the documentation to come up in IDEs (e.g., when you mouse over a method name) you need to have a separate comment on each function (i.e., `num_nodes` and `num_edges`).

Reply:
Fair enough.  I have improved the comments again.